x/pairing/keeper/unresponsive_provider.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/lavanet/lava/x/pairing/types"
 )
 
+const MINIMUM_PROVIDERS = 5
```

Comment:
seems like a waste to go over all the plans everytime we use this function, lets set a constant for this

Reply:
waste to go over a list of 4 values in order to prevent a mismatch? this const isn't enforced anywhere else and plans can change, do you want every proposal to check this const is updated?
you already have a mismatch with unitests

Reply:
https://github.com/lavanet/lava/pull/543/commits/1ad1bb568af8143d951d3d3282b8758615558e10